src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -684,6 +688,22 @@ std::string ClusterTaskManager::DebugStr() const {
   return buffer.str();
 }
 
+void ClusterTaskManager::RecordMetrics() {
+  stats::NumQueuedTasks.Record(metric_tasks_queued_);
+  stats::NumDispatchedTasks.Record(metric_tasks_dispatched_);
+  stats::NumSpilledTasks.Record(metric_tasks_spilled_);
```

Comment:
I think that normalizing the metrics to "per second" makes more sense given that these metrics won't always be recorded at exactly the same cadence. E.g. if the event loop queue or thread is saturated, you could have 5-10x increase in tick periods, meaning that the window for accumulating these metrics will be 5-10x larger, which may create false spikes in these metrics.

Reply:
Talked about this offline, and we're moving away from recording rates directly as gauges, and instead recording cumulative counts and doing after-the-fact `rate()` queries.